tests/fixture/bootstrapmonitor/README.md

```diff
@@ -48,9 +56,16 @@ and initiates a new test when one is found.
    - The network targeted by the test is determined by the value of
      the `AVAGO_NETWORK_NAME` env var set for the avalanchego
      container.
-   - Whether state sync is enabled is determined by the value of the
-     `AVAGO_CHAIN_CONFIG_CONTENT` env var set for the avalanchego
-     container.
+   - By default, the sync mode will be `full-sync`. The other sync
+     modes require providing additional configuration:
+     - The `only-p-chain-full-sync` mode is enabled by setting the
+       `AVAGO_PARTIAL_SYNC_PRIMARY_NETWORK` env var to `"true"` for
+       the avalanchego container. If this mode is enabled, the state
+       sync configuration for the C-Chain will be ignored.
+     - The `c-chain-state-sync` mode is enabled by including
+       `state-sync-enabled:false` in the
```

Comment:
This seems to suggest that state sync is the default. If that's the case, we may want to specify to enable full sync by setting `state-sync-enabled:false`, and not setting `only-p-chain-full-sync` option